ci-operator/step-registry/interop/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
```

Comment:
This OWNERS file needs to be updated to use the team name based on this conversation here
https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1677769587102299

Also we need PQE members that are responsible to maintain this scenario to be added as well.